okhttp/src/main/java/okhttp3/HttpUrl.java

```diff
@@ -842,6 +842,13 @@ public String fragment() {
   }
 
   /**
+   * Returns the redacted URL that would only show the domain name.
+   */
+  public HttpUrl redactedUrl() {
```

Comment:
Not sure this has any value being a `HttpUrl` since you can't really do anything with it. I would return `String` and just proactively `toString()` here.